### ISSUE TITLE
A11y_SQLAzureDataStudio_DeployDacpac_ReviewDeployPlan_ScreenReader: The control type and name for the table is not announced..

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9260]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.3.1;#JAWS;#NVDA;#MAS4.2.3;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Tool: Accessibility insights for desktop.
MAS References:MAS1.3.1, MAS4.2.3;

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select first radio button.
9. Navigate till next button & hit enter.
10. Provide all the details for Deploy Dacpac & hit next.
11. Launch screen reader NVDA.

**Actual:**
Issue1:
On navigating to the table using arrow keys there is no table name and control announced.

Issue2:
Screen reader does not read the data inside the table in a tabular structure there is no column row value and header cell associations.

**Expected:**
Issue1:
On navigating to the table using arrow keys  the table name and control should be announced.

Issue2:
The screen reader should read the data inside the table in a tabular structure with proper column row value and header cell associations.

**User Impact:**
Screen reader users will not receive the information in a tabular structure.

**Attachment link for Reference :**
[A11y_SQLAzureDataStudio_DeployDacpac_ReviewDeployPlan9260.zip](https://github.com/microsoft/azuredatastudio/files/4322638/A11y_SQLAzureDataStudio_DeployDacpac_ReviewDeployPlan9260.zip)

**Comment by alanrenmsft** [comment]
aria-label for the table component?

**Comment by kisantia** [comment]
Aria-label added for the table. Still need to fix issue 2. Might be because the table is in the loading component

**Comment by fsteffi** [comment]
hi @alanrenmsft  can you please share the sever name to connect through windows authentication to verify this issue 

GitHubTags:#Regressed:02-20-23;

**Comment by kisantia** [comment]
@fsteffi the type of authentication used doesn't matter for this scenario, so sql login can be used instead. If you need a sql login, please share you email and I can send you the info.

**Comment by fsteffi** [comment]
issue fixed on Version: 1.42.0-insider
Commit: c5fc37c373f965d77d3d860e7756fd9c4c4f0ca0
Date: 2023-02-17T06:29:22.173Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.25300

[9260-closed.webm](https://user-images.githubusercontent.com/24580570/220544655-b6558e48-b974-49f3-aae8-489225fb2292.webm)

GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
